.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-on: [push]
+on: [push, pull_request]
```

Comment:
This should probably only be on `pull_request`. Else you get a double event triggering for each commit, which can be really noisy.